src/plots/gl3d/scene.js

```diff
@@ -174,6 +174,7 @@ function initializeGLPlot(scene, fullLayout, canvas, gl) {
     var relayoutCallback = function(scene) {
         var update = {};
         update[scene.id] = getLayoutCamera(scene.camera);
+        scene.saveCamera(scene.graphDiv.layout);
```

Comment:
I'm _ok_ with this here.

 **BUT** we should a comment in the [`resetCameraLastSave`](https://github.com/plotly/plotly.js/blob/d8e27dd33270aa7e4bb01a612cc99c51fa3cb68a/src/components/modebar/buttons.js#L353) mode bar button handler - which looks in the un-updated `fullLayout` to reset the camera to the last _saved_ position.